xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml

```diff
@@ -42,12 +42,12 @@
   <content>invitation.applicationsPanelEntryLabel=Invitation
 
 # Invitation Administration Section
-Invitation.WebHome_from_address.hint=The email address used to send invitation emails from.
-Invitation.WebHome_smtp_server_password.hint=The password to connect to the SMTP server, only when authentication is required.
-Invitation.WebHome_smtp_server_username.hint=The user name to connect to the SMTP server, only when authentication is required.
+Invitation.WebHome_from_address.hint=The email address used to send the invitation emails.
+Invitation.WebHome_smtp_server_password.hint=The password to connect to the SMTP server.
+Invitation.WebHome_smtp_server_username.hint=The username to connect to the SMTP server.
 Invitation.WebHome_smtp_port.hint=The port of the SMTP server to use.
 Invitation.WebHome_smtp_server.hint=The SMTP server to send to.
-Invitation.WebHome_javamail_extra_props.hint=The list of additional Java Mail properties to use.</content>
+Invitation.WebHome_javamail_extra_props.hint=List of additional JavaMail properties to use.</content>
```

Comment:
Please keep the `The` start, to be consistent with the previous hints.

Reply:
"The" before list seems wrong. For eg. the title of this (and all other) Wikipedia page starts directly from list : https://en.m.wikipedia.org/wiki/List_of_Prime_Ministers_of_India

Reply:
Done.